pkg/providers/aws/cluster_vpc.go

```diff
@@ -459,8 +460,32 @@ func getSubnets(ec2Svc ec2iface.EC2API) ([]*ec2.Subnet, error) {
 	return subs, nil
 }
 
+func getRHMIVpc(ctx context.Context, c client.Client, ec2Svc ec2iface.EC2API, logger *logrus.Entry, clusterID, organizationTag string) (*ec2.Vpc, error) {
+	// get vpcs
+	vpcs, err := ec2Svc.DescribeVpcs(&ec2.DescribeVpcsInput{})
+	if err != nil {
+		return nil, errorUtil.Wrap(err, "error getting subnets")
+	}
+
+	//there will be a tags associated with the seperate vpc
+	//Name='RHMI Cloud Resource VPC' and integreatly.org/clusterID=<infrastructure-id>
+	//use these in order to pick up the correct one.
+	// find associated vpc to tag
+	var foundVPC *ec2.Vpc
+	for _, vpc := range vpcs.Vpcs {
+		for _, tag := range vpc.Tags {
+			if (*tag.Key == fmt.Sprintf("%sclusterID", organizationTag) && *tag.Value == clusterID) ||
+				(*tag.Key == DefaultRHMIVpcNameTagKey && *tag.Value == DefaultRHMIVpcNameTagValue) {
```

Comment:
The `Name` tag on the VPC will be the same for all clusters. If there's a leftover VPC or we eventually move to supporting multi-cluster AWS accounts, this will return `true` on the wrong VPC due to the `or`. I think we should ignore the name tag (It's mainly for SRE/people looking in the console) and focus on the cluster ID tag